pkg/template/static_context.go

```diff
@@ -584,6 +591,61 @@ func (ctx StaticCtx) yamlEscape(plain string) string {
 	return indented
 }
 
+func (ctx StaticCtx) kubernetesVersion() string {
+	clientset, err := ctx.getClientset()
+	if err != nil {
+		return ""
+	}
+	sv, err := getK8sServerVersion(clientset)
+	if err != nil {
+		return ""
+	}
+	return strings.TrimPrefix(sv.GitVersion, "v")
```

Comment:
Any particular reason for trimming `v`?

Reply:
Just to make sure it works with with sprig semver comparison. Apparently, that's what we do for the KOTS version template function. It might work with the "v", but I wanted to keep things consistent.